esptfa_arima/Test_Management/permissions/permissions.py

```
@@ -0,0 +1,10 @@
+
+
+# permissions for checking if user is a teacher
+from rest_framework import permissions
+
+class IsTeacher(permissions.BasePermission):
+    def has_permission(self, request, view):
```

Ruff (0.14.10)

Unused method argument: `view`

(ARG002)